src/lib/components/scrollbarwrapper/ScrollbarWrapper.component.tsx

```diff
@@ -3,7 +3,8 @@ import styled, { css } from 'styled-components';
 
 import { getTheme } from '../../utils';
 type Props = {
-  children: JSX.Element;
+  children: JSX.Element | JSX.Element[];
+  theme: { [key: string]: string };
```

Comment:
I just discovered the scrollbar color defined in core-ui were used degraded color. 
I will let you update it based on the Figma design here:
https://www.figma.com/file/N2jOAK44dTSiBclDjXzz6a/ARTESCA?node-id=2622%3A893